runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -212,6 +213,12 @@ final class GrpcClientSettings private (
   def withConnectionAttempts(value: Int): GrpcClientSettings =
     copy(connectionAttempts = Some(value))
 
+  /**
+    * To override any default channel configurations used by netty. Only for power users.
+    */
```

Comment:
Oh, the build failure is because this comment is not formatted according to our current style guide: the last 2 lines should go 1 character to the left. Compiling from sbt (or running `sbt scalariformFormat`) should do this automatically for you. We'll look into making that more obvious.

Reply:
@raboof thanks for the tip, was wondering why it failed too